lib/mocha.js

```diff
@@ -471,7 +491,14 @@ Mocha.prototype.run = function(fn) {
   var options = this.options;
   options.files = this.files;
   var runner = new exports.Runner(suite, options.delay);
-  var reporter = new this._reporter(runner, options);
+
+  // For each loaded reporter constructor, create
+  // an instance and initialize it with the runner
+  var reporters = this._reporters.map(function(reporterConfig) {
+    var _reporter = reporterConfig.fn;
+    options.reporterOptions = reporterConfig.options;
```

Comment:
Runners may not clone the passed in options, and instead hold a reference, and even modify its content. I think it would be better to clone the `options` object in each loop, and pass that in to the constructor.